### PR TITLE
Look up images when creating PR job

### PR DESCRIPTION
**Pull request by petey**
When creating a PR, we need to look up the screwdriver.yaml in the PR for the main job, in case the images have changed.

### CODE REVIEW

**Comment by nkatzman** [comment]
👍

File: plugins/webhooks/github.js [hunk 1]
```diff
@@ -24,9 +24,14 @@ function pullRequestOpened(options, request, reply) {
     const name = options.name;
     const sha = options.sha;
     const username = options.username;
+    const ref = options.prRef;
```
Comment (d2lam) [inline plugins/webhooks/github.js]: Add these to comment section too

**Comment by d2lam** [comment]
Small comment. The rest looks good to me 👍

**Comment by FenrirUnbound** [comment]
👍 

Looks good as long as the outstanding comments are addressed

ugomeuaofqgx

**Pull request merged**